### PR TITLE
Add wrapper for DF driver MPU6050

### PR DESCRIPTION
**Pull request by eyeam3**
Adds the DF wrapper for the MPU6050 IMU sensor. Furthermore, it enables the sensors app for the Parrot Bebop and updates the `mainapp.config` accordingly.

### CODE REVIEW

**Comment by bkueng** [comment]
@eyeam3 nice code! There are quite a few TODO's in there. Are you planning to fix these?

File: cmake/configs/posix_bebop_default.cmake [hunk 1]
```diff
@@ -99,4 +101,5 @@ set(config_module_list
 
 set(config_df_driver_list
   ms5607
+  mpu6050
```
Comment (julianoes) [inline cmake/configs/posix_bebop_default.cmake]: nitpick: indenting is a bit weird. I'd prefer a tab.

File: posix-configs/bebop/mainapp.config [hunk 2]
```diff
@@ -4,9 +4,10 @@ param load
...
 param set MAV_BROADCAST 1
 df_ms5607_wrapper start
+df_mpu6050_wrapper start
+sensors start
 sleep 1
 mavlink start -u 14556 -r 1000000
 sleep 1
-mavlink stream -u 14556 -s ATTITUDE -r 50
-mavlink stream -u 14556 -s ALTITUDE -r 1
+mavlink stream -u 14556 -s HIGHRES_IMU -r 50
```
Comment (julianoes) [inline posix-configs/bebop/mainapp.config]: You don't need `ATTITUDE` anymore?

File: src/platforms/posix/drivers/df_mpu6050_wrapper/df_mpu6050_wrapper.cpp [hunk 3]
```diff
@@ -0,0 +1,712 @@
...
+				    integral_dt_unused);
+
+	// If we are not receiving the last sample from the FIFO buffer yet, let's stop here
+	// and wait for more packets.
+	if (!data.is_last_fifo_sample) {
+		return 0;
+	}
+
+	// The driver empties the FIFO buffer at 1kHz, however we only need to publish at 250Hz.
+	// Therefore, only publish every forth time.
```
Comment (julianoes) [inline src/platforms/posix/drivers/df_mpu6050_wrapper/df_mpu6050_wrapper.cpp]: Is this comment correct for your implementation over I2C? I doubt it. And is the publish rate sufficient?

File: posix-configs/bebop/mainapp.config [hunk 4]
```diff
@@ -4,9 +4,10 @@ param load
 param set SYS_AUTOSTART 4001
 param set MAV_BROADCAST 1
 df_ms5607_wrapper start
+df_mpu6050_wrapper start
```
Comment (julianoes) [inline posix-configs/bebop/mainapp.config]: Is the rotation correct on the Bebop?

File: posix-configs/bebop/mainapp.config (hunk 2 above)
Comment (eyeam3) [inline posix-configs/bebop/mainapp.config]: The EKF is not running at the moment and not tested for the Bebop. So, there is no attitude published at the moment. I removed it for now and will add it back, when it is tested.

File: src/platforms/posix/drivers/df_mpu6050_wrapper/df_mpu6050_wrapper.cpp (hunk 3 above)
Comment (eyeam3) [inline src/platforms/posix/drivers/df_mpu6050_wrapper/df_mpu6050_wrapper.cpp]: The sampling on the device itself happens with 1kHz and written into a FIFO queue and the driver's measurement interval is 1000 us. So far I didn't have any issues with FIFO overflows or corrupted data and the measurements are reasonable.
Should I test something else?

File: posix-configs/bebop/mainapp.config (hunk 4 above)
Comment (eyeam3) [inline posix-configs/bebop/mainapp.config]: No, it is not. I prepared a fix within another future pull-request. But I will add it here.

**Comment by eyeam3** [comment]
I did not fix the TODOs because some of them require bigger adjustments and I would try to get a running firmware for the Bebop platform first. The missing implementations are not absolutely necessary for the driver and the basic functionality is given.

File: src/platforms/posix/drivers/df_mpu6050_wrapper/df_mpu6050_wrapper.cpp (hunk 3 above)
Comment (julianoes) [inline src/platforms/posix/drivers/df_mpu6050_wrapper/df_mpu6050_wrapper.cpp]: Ah sorry, I was just assuming I2C to be slower but seems fast enough then, nevermind.

**Comment by julianoes** [comment]
Rebased and applied, thanks!

**Pull request closed without merging**